### ISSUE TITLE
Can not donload Training dataset from Google Drive

### ISSUE THREAD

**Issue by herckman77** [yinanhe/ForgeryNet#50]
Hi and thank you for your work
When i try to download the training and validation dataset using the link you proivde in your site 'https://yinanhe.github.io/projects/forgerynet.html#  i am not redirected to Google Drive butt at a page that flickers and nothing happens. I have tried in 3 different browsers and 2 different computers. Can you please check if the link is working?
training set link 'https://opendatalab.com/ForgeryNet/download
validation set link 'https://opendatalab.com/ForgeryNet/download'
I am planning to use the dataset for my Master Thesis
Thank you.

**Comment by yinanhe** [comment]
There are a few small changes, you can access it directly through this link.
https://opendatalab.com/OpenDataLab/ForgeryNet

**Comment by yzheng97** [comment]
Thanks for your great work. I have applied for the CDK/CLI access to the dataset. May I ask how soon can my application be approved?

> There are a few small changes, you can access it directly through this link. https://opendatalab.com/OpenDataLab/ForgeryNet

**Comment by The-Emperor-0** [comment]
Hello, can I still download the code for the dataset now? My download request was submitted for a week and is still under review.
